libs/wallet-ui/src/components/interactions/flows/transaction-review/transaction-review.tsx

```diff
@@ -154,10 +154,8 @@ export const TransactionReview = ({
                 </span>
               </CopyWithTooltip>
             )}
-            {data.transaction.txHash && explorerUrl && (
-              <ExternalLink
-                href={`${explorerUrl}/txs/${data.transaction.txHash}`}
-              >
+            {txUrl(data.transaction.txHash) && (
```

Comment:
suggestion (non-blocking): i think if you'd clean this up and saved it to a variable would look a lot cleaner, like:

```
const { getTxUrl } = useExplorerLinks()
const txUrl = getTxUrl(data.transaction.txHash)

{txUrl && <ExternalLink href={txUrl} />}
```